eradiate/experiments/_rami.py

```diff
@@ -147,6 +147,7 @@ def _normalize_measures(self) -> None:
                             xmax=0.5 * self.canopy.size[0],
                             ymin=-0.5 * self.canopy.size[1],
                             ymax=0.5 * self.canopy.size[1],
+                            z=self.canopy.size[2],
```

Comment:
Good point. I don't remember: is the fact that we target the ToC mentioned in the documentation?